bundle/bundle-writer.go

```diff
@@ -382,15 +389,18 @@ func addDatabase(manifestFile *ManifestBundle, databases []common.Database) (*Ma
 	for _, db := range databases {
 		switch db.Name {
 		case "mysql":
-			release.Name = db.Name
+			release.ChartName = db.Name
+			release.DisplayName = "starter-" + db.Name
```

Comment:
Shouldn't include `starter-` at the beginning of the chart display name.

Reply:
done!